java/src/jmri/jmrix/sprog/SprogConstants.java

```diff
@@ -26,6 +26,16 @@
     /* Maximum number of slots for soft command station */
     public final static int MAX_SLOTS = 16;
 
+    /* Worst case DCC packet is ~10 ms, which equates to 100 packets/s. Wait for
+     * a somwhat arbitrary time before reporting a possible issue. A delay of
+     * 33 ms equates to 30 packets/s if sustained. */
+    public final static int PACKET_DELAY_WARN_THRESHOLD = 33;
```

Comment:
No need to change this, but if you remove the "final" it can be set via a script if somebody wants to customize it.

Reply:
@bobjacobsen That's a great idea, as The timeouts are causing me headaches, esp., on Raspberry Pi, and some older Pi-SPROG firmware versions. I will change it. You've got me thinking, what else could be changed :)